docs/website/docs/command-reference/dev.md

```diff
@@ -417,13 +417,9 @@ This state file contains the forwarded ports:
 
 ## Env File
 
-When `odo dev` is executed, a `.odo/env/env.yaml` file is created if it does not exist. It stores the namespace/project that will be used by the component, and its default value is set to currently active namespace/project.
+When `odo dev` is executed, a `.odo/env/env.yaml` file is created if it does not exist. It stores the namespace/project that will be used by the component during the dev session. The value is removed from the file at the end of the session.
```

Comment:
Not meaning to make this change here, but this seems like a good candidate to be put into `devstate.json` instead. WDYT?

Reply:
Yes, I think so. @kadel what's your opinion?